driver-core/src/test/functional/com/mongodb/client/model/search/AggregatesSearchIntegrationTest.java

```diff
@@ -183,24 +201,105 @@
     private static Map<MongoNamespace, CollectionHelper<BsonDocument>> collectionHelpers;
 
     @BeforeAll
     static void beforeAll() {
         collectionHelpers = new HashMap<>();
         collectionHelpers.put(MFLIX_MOVIES_NS, new CollectionHelper<>(new BsonDocumentCodec(), MFLIX_MOVIES_NS));
+        collectionHelpers.put(MFLIX_EMBEDDED_MOVIES_NS, new CollectionHelper<>(new BsonDocumentCodec(), MFLIX_EMBEDDED_MOVIES_NS));
         collectionHelpers.put(AIRBNB_LISTINGS_AND_REVIEWS_NS, new CollectionHelper<>(new BsonDocumentCodec(), AIRBNB_LISTINGS_AND_REVIEWS_NS));
     }
 
     @BeforeEach
     void beforeEach() {
         assumeTrue(isAtlasSearchTest());
+        assumeTrue(serverVersionAtLeast(7, 1));
```

Comment:
Is this intended to be the case for all tests, or just the new vector search tests?

Reply:
Thanks, it's not. Done.